org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/debug/firefox/RunFirefoxDebugTab.java

```diff
@@ -30,12 +30,11 @@ public RunFirefoxDebugTab() {
 	public void createControl(Composite parent) {
 		super.createControl(parent);
 		reloadOnChange = new Button(resComposite, SWT.CHECK);
-		reloadOnChange.setText("Reload on change");
+		reloadOnChange.setText("Reload on change"); //$NON-NLS-1$
```

Comment:
This will need to be internationalized, so let's extract it to some `Messages` class and *not* mark it as `//$NON-NLS-1`. NON-NLS means that a string shouldn't be translated, this one should.

Reply:
Ok, will externalize.